### ISSUE TITLE
Push to SuaveIO org repo

### ISSUE THREAD

**Issue by haf** [OlegZee/Suave.OAuth#4]
Could you, please? :)

**Comment by OlegZee** [comment]
Of course I will!

@haf Could you please explain what should I do with project to meet Suave standarts? I'd like to migrate it to new .net sdk and provide netcore and net451 binaries.

**Comment by haf** [comment]
- Build for .Net Core
 - .Net:
   - Build for .Net
   - Build for Mono
 - Unit and integration test with Expecto – always consider how Suave.Testing can be improved.
 - Build nuget for both of the above
 - Merge nugets for both of the above
 - Push nuget

We're using Travis and Appveyor.

**Comment by OlegZee** [comment]
@haf I need permissions to push to suaveio repo

**Comment by haf** [comment]
@OlegZee Of course; I've made you admin of that repository now. Thank you for getting involved!

**Comment by OlegZee** [comment]
@haf I pushed new stuff to netcore branch and pushed 0.9.0-pre2 nuget package.
It contains both netstandard16 and net461 binaries and works fine in projects targeting full framework.

I added two separate samples, one for netcore and one for net4.6.1 both are ok.
Please let me know if the following tasks are still making sense:

 * Build for .Net
 * Build for Mono
 * Merge nugets for both of the above

I will keep working on updated CI/build scripts. No idea how to make unit tests.
Btw, is it ok to use build scripts made on [homegrown](https://github.com/OlegZee/Xake/wiki)?

**Comment by haf** [comment]
Xake looks nice. Let me get back to you re the rest.

**Comment by haf** [comment]
Yes, that makes very much sense. Go for it!
